interfaces/builtin/serial_port.go

```diff
@@ -154,7 +154,8 @@ func (iface *SerialPortInterface) ConnectedPlugSnippet(plug *interfaces.Plug, sl
 	switch securitySystem {
 	case interfaces.SecurityAppArmor:
 		if iface.hasUsbAttrs(slot) {
-			// This apparmor rule must match serialDeviceNodePattern
+			// This apparmor rule is an approximation of serialDeviceNodePattern
+			// specifically AARE does not support the '+' token for "one or more"
```

Comment:
AARE's '*' also isn't the same as regex. I would just leave that part of the comment out. If you really wanted something, use this instead:
```
// This apparmor rule is an approximation of serialDeviceNodePattern
// (AARE is different than regex, so we must approximate).
```